examples/v2_boolean_org_enforce/main.tf

```
@@ -29,3 +29,17 @@ module "gcp_org_policy_v2" {
   constraint  = "compute.requireOsLogin"
   policy_type = "boolean"
 }
+
+module "parameterized_org_policy_v2" {
+  source  = "terraform-google-modules/org-policy/google//modules/org_policy_v2"
+  version = "~> 5.0"
```

nit: let's bump in advance as we know this will go into v7.0

```suggestion
  version = "~> 7.0"
```

---

done